src/assets/svg/icons/other/Banner.tsx

```diff
@@ -5,8 +5,7 @@ function SvgBanner(props: SvgProps) {
   return (
     <Svg width={20} height={20} fill="black" viewBox="0 0 320 160" {...props}>
       <Path
-        fill="white"
-        opacity={0.2}
+        fill="#AAE6EC"
```

Comment:
Was this change intended?

Reply:
Sort of. The SVG-components under `src/assets` are generated by `yarn generate-svgs`, which uses the SVGs from `assets` under the root-folder as source. In other words, these are auto-generated and shouldn't really be modified directly. I've added a commit which adjusts Banner-opacity from DisappearingHeader instead